src/commonMain/kotlin/trace-consumer.kt

```diff
@@ -1,9 +1,11 @@
 package kotlinx.html.consumers
 
 import kotlinx.html.*
+import kotlin.random.*
 
 class TraceConsumer<R>(val downstream: TagConsumer<R>, val println: (String) -> Unit) : TagConsumer<R> by downstream {
-    private val id = "ID-${currentTimeMillis() % 16384}"
+    private val random = Random.nextInt()
```

Comment:
Decided to not provide an option to set this `Random` as parameter (eg for tests) because the current implementation using `currentTimeMillis` isn't mockable too. If wanted, we could add an optional `Random` parameter too.

Reply:
I would avoid using random here because it could have collision in rare cases and it can be interesting to debug :). Could you keep time or try using UUID instead?

Reply:
I think, we also can use identity hashcode here

Reply:
I see, there is no official multiplatform kotlinx.uuid (yet, only the one of Sergey I forked), so I switched to `hashCode()`